sites/avivator/src/utils.js

```diff
@@ -102,7 +102,7 @@ export async function createLoader(
       }
       const url = urlOrFile;
       const res = await fetch(url.replace(/ome\.tif(f?)/gi, 'offsets.json'));
-      const isOffsets404 = res.status === 404;
+      const isOffsets404 = res.status !== 200;
```

Comment:
So let's change this variable name then too.

Reply:
i just changed it to isOffsetsNot200, hope it is ok.thanks